### PR TITLE
fix the error and added bitbucket api call

### PR DESCRIPTION
**Pull request by ajayVibinex**

### CODE REVIEW

**Comment by ajayVibinex** [comment]
like in github we're showing the the highlighted files in pr, so it was not happening for bitBucket. So fixed it by calling api for bitBucket.

File: scripts/contentScript.js [hunk 1]
```diff
@@ -245,48 +245,26 @@ function addCssElementToBitbucket(highlightedPRIds, userId) {
...
-	chrome.storage.sync.get(["backendUrl"]).then(async ({ backendUrl }) => {
-		const body = {
-			"repo_owner": repoOwner,
-			"repo_name": repoName,
-			"user_id": userId,
-			"is_github": true
-		}
-		const url = `${backendUrl}/relevance/pr`;
-		const highlightedPRIds = await apiCall(url, body);
+function getHighlightedPR(highlightedPRIds) {
```
Comment (avikalpg) [inline scripts/contentScript.js:248]:
```suggestion
function highlightedRelevantPRs(highlightedPRIds) {
```
Change the name of the function based on the changed actions within it.

File: scripts/contentScript.js [hunk 2]
```diff
@@ -313,6 +290,7 @@ const orchestrator = (tab_url, websiteUrl, userId) => {
 		console.warn(`[Vibinex] You are not logged in. Head to ${websiteUrl} to log in`);
 		// TODO: create a UI element on the screen with CTA to login to Vibinex
 	}
+	chrome.storage.sync.get(["backendUrl"]).then(async ({ backendUrl }) => {
 	if (urlObj[2] == 'github.com') {
```
Comment (avikalpg) [inline scripts/contentScript.js:294]: change the indentation of the child block accordingly.

**Review by avikalpg — APPROVED** [review]
I have two minor comments. Please incorporate them before merging.

**Review by avikalpg — APPROVED** [review]

**Pull request merged**